rpm/tarantool-http.spec

```diff
@@ -20,14 +20,19 @@ This package provides a HTTP server for Tarantool.
 %setup -q -n %{name}-%{version}
```

Comment:
We should bump epoch due to #134. If it is solved somehow, I don't see how. Please, clarify.

Reply:
Epoch was already bumped [here](https://github.com/tarantool/http/commit/4db9aa3a16a767fd82b2125984af16c262b3ec6b#diff-c0279f62819ca56bac5a167034ae765318b020d36c8b151f8d999ba064c5ee39R4) and as we tested, the proper version of the package is installed.